cli/tests/e2e/snapshots/test_output/test_debug_experimental_rule/results.txt

```diff
@@ -1,5 +1,5 @@
 === command
-SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS_FILE="<MASKED>" SEMGREP_VERSION_CACHE_PATH="<MASKED>" SEMGREP_ENABLE_VERSION_CHECK="0" SEMGREP_SEND_METRICS="off" semgrep --debug --strict --config rules/experiment/experiment.yaml targets/experiment/experiment.py
+SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS_FILE="<MASKED>" SEMGREP_VERSION_CACHE_PATH="<MASKED>" SEMGREP_ENABLE_VERSION_CHECK="0" SEMGREP_SEND_METRICS="off" semgrep --debug --strict --config <MASKED> <MASKED>
```

Comment:
why the mask here?

Reply:
Does it hurt? There were a bunch of very specific masking rules. I replaced them by a simpler one that masks all paths because it's easier to maintain.